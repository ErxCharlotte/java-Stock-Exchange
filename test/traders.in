TRADERS
ADD trader1 100.00
ADD trader2 100.00
ORDER 0000
IMPORT trader1 ABC 100.00
IMPORT trader1 CDE 500.00
SELL trader1 ABC 100 5
TRADERS
EXIT