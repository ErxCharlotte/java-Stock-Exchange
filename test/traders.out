$ No traders in the market.
$ Success.
$ Success.
$ No orders in either book in the market.
$ Trader now has 100.00 units of ABC.
$ Trader now has 500.00 units of CDE.
$ No trades could be made, order added to sell book.
$ trader1
trader2
$ Have a nice day.